src/main/java/org/graylog/plugins/threatintel/functions/otx/OTXLookupResult.java

```diff
@@ -19,22 +19,33 @@
 import com.google.common.base.Joiner;
 import com.google.common.collect.ForwardingMap;
 import com.google.common.collect.ImmutableMap;
+import org.graylog2.plugin.lookup.LookupResult;
 
 import java.util.Map;
 
 public class OTXLookupResult extends ForwardingMap<String, Object> {
 
+    public static final String LOOKUP_KEY = "key";
+    public static final String MESSAGE = "message";
+    public static final String HAS_ERROR = "has_error";
+    public static final String OTX_THREAT_INDICATED = "otx_threat_indicated";
     private final ImmutableMap<String, Object> results;
 
-    public static final OTXLookupResult EMPTY = new EmptyOTXLookupResult();
-    public static final OTXLookupResult FALSE = new FalseOTXLookupResult();
+    protected static final OTXLookupResult EMPTY = new EmptyOTXLookupResult();
+    protected static final OTXLookupResult FALSE = new FalseOTXLookupResult();
+
+    public static OTXLookupResult buildFromError(LookupResult lookupResult) {
+        return new FalseOTXLookupResult(
+                (String) lookupResult.multiValue().get(LOOKUP_KEY),
+                (String) lookupResult.multiValue().get(MESSAGE));
+    }
 
     public static OTXLookupResult buildFromIntel(OTXIntel intel) {
```

Comment:
It seems that this method is no longer used.

Reply:
good catch